pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3144,27 +3144,14 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			})
 		}
 
-		fakeHotPlugRequest := func(vmi *virtv1.VirtualMachineInstance, addOpts []AddInterfaceOptions) {
-			for _, req := range addOpts {
-				vmi.Spec.Networks = append(
-					vmi.Spec.Networks,
-					virtv1.Network{
-						Name: req.Name,
-						NetworkSource: virtv1.NetworkSource{
-							Multus: &virtv1.MultusNetwork{NetworkName: req.NetworkAttachmentDefinitionName},
-						},
-					})
-			}
-		}
-
 		const (
 			firstVMInterface  = "oldIface1"
 			firstVMNetwork    = "oldnet1"
 			secondVMInterface = "oldiface2"
 			secondVMNetwork   = "oldnet2"
 		)
 
-		Context("k8s API is down - i.e. you cannot update the pod status", func() {
+		Context("k8s API is down", func() {
```

Comment:
Is there a chance you could extract the refactoring changes to another commit?

Reply:
There was no refactoring, just added the new logic. And fixed the `controller.updateMultusAnnotation` test which are all in this file.